optimade/server/main.py

```diff
@@ -116,7 +117,9 @@ def general_exception_handler(request: Request, exc: Exception):
     tags=["Structure"],
 )
 def get_structures(request: Request, params: EntryListingQueryParams = Depends()):
-    return u.get_entries(structures, StructureResponseMany, request, params)
+    return u.get_entries(
```

Comment:
Minor thing, but let's have `utils.py` import `entry_collections` instead of passing it through as a parameter?
If there is an import-loop problem, I guess this is fine for now.